leaf-ui/js/onFunctions.js

```diff
@@ -238,9 +238,9 @@ function switchToModellingMode() {
     $('.analysis-clears').css("display", "none");
     
     // Appear 
-    $('#stencil').css("display","");
-    $('#btn-view-assignment').css("display","");
     $('#btn-view-intermediate').css("display","");
+    $('#btn-view-assignment').css("display","");
```

Comment:
Is there a particular reason why you changed the order?

Reply:
No, I think I was experimenting. I changed it back to the original order.